### PR TITLE
Add investor account update  page

### PR DESCRIPTION
**Pull request by barbara-chaves**
This PR adds a page to the investors edit theirs profiles
AND refactors the investor creation form to use the same component for create and update the investor profile

## Testing instructions

Acceptance criteria

There must be a way for ”Investors” to edit their account information

The functionality to edit Investor accounts will be at the Investor’s “My area”  

These are the fields that can be edited: See [reqs doc for specifications](https://docs.google.com/spreadsheets/d/1qtP-qkm8iS6OlluiJwRvQoi6rCns9iKhRYoNzc9qkGg/edit#gid=1884416048)

There will be a way to validate the information entered in the fields

Errors are displayed if the information is incorrect

There will be a way to confirm or dismiss the edition of the account

Note: Save/cancel

The layout must comply with the designs defined at [HeCo - Work in progress](https://www.figma.com/proto/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=1326%3A21536&viewport=396%2C48%2C0.06&scaling=scale-down&starting-point-node-id=1326%3A21536&show-proto-sidebar=1)

Note: language is not editable

Any member of the account can edit

Edition of an account doesn’t have an impact on the account’s status
## Tracking

[LET-608](https://vizzuality.atlassian.net/browse/LET-608)

### CODE REVIEW

**Comment by anamontiaga** [comment]
@barbara-chaves how can i test it from de user experience?

File: frontend/pages/investors/edit.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  enums,
+}) => {
+  const { formatMessage } = useIntl();
+  const { push } = useRouter();
+
+  const updateInvestor = useUpdateInvestor();
+
+  const handleOnComplete = () => {
+    push(Paths.Dashboard);
+  };
```
Comment (SARodrigues) [inline frontend/pages/investors/edit.tsx:45]:
```suggestion
  const handleOnComplete = () => {
    router.push((router.query?.returnPath as string) || Paths.Dashboard);
  };
```

File: frontend/pages/investors/edit.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+type EditInvestorServerSideProps = {
+  enums: GroupedEnums;
+};
+
+const EditInvestorPage: PageComponent<EditInvestorServerSideProps, FormPageLayoutProps> = ({
+  enums,
+}) => {
+  const { formatMessage } = useIntl();
+  const { push } = useRouter();
```
Comment (SARodrigues) [inline frontend/pages/investors/edit.tsx:39]:
```suggestion
}) => {
  const router = useRouter();
  const { formatMessage } = useIntl();
```

File: frontend/pages/investors/new.tsx [hunk 3]
```diff
@@ -51,128 +35,27 @@ type NewInvestorServerSideProps = {
...
-    await handleSubmit(onSubmit)();
-    if (!errors) {
-      setCurrentPage(currentPage + 1);
-    }
-  };
-
-  const getPageErrors = (page: number) => {
-    return formPageInputs[page].some((input) => errors.hasOwnProperty(input));
+  const handleOnComplete = () => {
+    push(Paths.Dashboard);
```
Comment (SARodrigues) [inline frontend/pages/investors/new.tsx:44]:
```suggestion
  const handleOnComplete = () => {
    router.push((router.query?.returnPath as string) || Paths.Dashboard);
```

File: frontend/pages/investors/new.tsx [hunk 4]
```diff
@@ -51,128 +35,27 @@ type NewInvestorServerSideProps = {
 const NewInvestorPage: PageComponent<NewInvestorServerSideProps, FormPageLayoutProps> = ({
   enums,
 }) => {
-  const intl = useIntl();
   const { formatMessage } = useIntl();
   const { push } = useRouter();
```
Comment (SARodrigues) [inline frontend/pages/investors/new.tsx:39]:
```suggestion
  const router = useRouter();
  const { formatMessage } = useIntl();
```

**Review by SARodrigues — APPROVED** [review]
Looks great! 

The suggestion changes are to make it so that the user will be returned to the previous page after form completion. 

Might be worth testing them; though they're made the same way the project developer ones are (the respective dashboard button supports it already).

**Comment by barbara-chaves** [comment]
> @barbara-chaves how can i test it from de user experience?

You have to login as a Investor or Project developer and go to 
dashboard/account-information

**Pull request merged**